package/MDAnalysis/converters/RDKit.py

```diff
@@ -475,30 +487,68 @@ def _add_mda_attr_to_rdkit(attr, value, mi):
     if attr == "names":
         # RDKit needs the name to be properly formatted for a
         # PDB file (1 letter elements start at col 14)
-        name = re.findall(r'(\D+|\d+)', value)
+        name = re.findall(r'(\D+|\d+)', value)  # splits alphabet from numeric
+        # alpha-numeric name
         if len(name) == 2:
-            symbol, number = name
-            if len(number) > 2 and len(symbol) == 1:
-                value = "{}{}".format(symbol, number)
+            # only two characters
+            if len(value) == 2:
+                value = f" {value} "
+            # more than two --> cut to 4 characters
             else:
-                value = "{:>2.2}{:<2.2}".format(symbol, number)
+                symbol, number = name
+                value = f"{symbol + number:>4.4}"
+        # no number in the name
         else:
-            # no number in the name
-            value = " {:<}".format(name[0])
+            value = f" {name[0]:<3.3}"
```

Comment:
I'm guessing this is for RDKit's PDB writer, but it's IMO suboptimal to edit the names like this. It'd be nice to set another property with `SetProp` without this character limit, if this property is used for conversion from RDKit into MDAnalysis as well.

Reply:
I might also have missed sometihng but I don't fully understand the left 3-column justification if there's no number in the name -- is this an RDKit quirk? Otherwise you could maybe just use the `center` function: `f"{value.center(4):4.4}"`